Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -101,16 +113,27 @@ private bool scaleActive
         private bool affordancesVisible = false;
         private float runningAverageSmoothing = 0.0f;
         private const float percentToDecimal = 0.01f;
+        private Material currentMaterial;
         private List<Vector2> unTransformedUVs = new List<Vector2>();
         private Dictionary<uint, HandPanData> handDataMap = new Dictionary<uint, HandPanData>();
         private List<IMixedRealityHandPanHandler> handlerInterfaces = new List<IMixedRealityHandPanHandler>();
         #endregion Private Properties
 
+        /// <summary>
+        /// The Reset function sets the pan and zoom back to their starting settings.
```

Comment:
```suggestion
        /// Sets the pan and zoom back to their starting settings.
```

Reply:
Nit if only because in the doccomment for a function it's not really necessary to re-name the function name itself.

Reply:
will do